### DISCUSSION TITLE
Publish to Twitter automatically (create a tweet)

### DISCUSSION THREAD

**Discussion by Vadorequest** [transitive-bullshit/nextjs-notion-starter-kit#263] [Ideas]
I wonder if adding the ability to automatically publish articles to Twitter, through the creation of a Tweet, would be a useful feature.

More generally speaking, such a publication system could be added for other mediums, such as: Medium, Dev.to, etc.

I think this could be done with Zapier automations and wouldn't necessarily request code changes from this repo, although we might want to adapt the documentation to point toward those solutions.

Is that something that people would find useful? I personally use Dev.to and Medium and Twitter, and would like to publish there.
One way to go would be to have a "Publish to Twitter" checkbox on the Notion Table. I haven't checked the actual feasibility but I feel like it's doable.

**Comment by transitive-bullshit** [comment]
I think this would be really awesome.

At the same time, I don't think it falls within the scope of this project.

If this existed and it "just worked", I'd certainly consider using it... but getting it to just work for the nuances of these different platforms is a difficult task. You would, for instance, need to convert tables appearing in an article into an image or some format that could be embedded as generic markdown. Super interesting for sure, but well outside the scope of this project.

This project by @dragonman225 may be useful if you're going down this path: https://github.com/dragonman225/nast

> I think this could be done with Zapier automations and wouldn't necessarily request code changes from this repo, although we might want to adapt the documentation to point toward those solutions.

Agreed. If you implement an automated solution and publish an accompanying guide, I think that would be the best way to make things more concrete. I'd be happy to link to it from the readme.

**Reply by transitive-bullshit** [reply]
Also check out https://www.queue.so by @phuctm97

**Reply by Vadorequest** [reply]
Thanks for the resources, I'll definitely check them out if I decide to go down that rabbit hole!
